pytorch_toolkit/image_classification/model_templates/custom-classification/README.md

```diff
@@ -1,15 +1,80 @@
 # Image classification
 
-Models that are able to classify images.
+Performance results pivot table:
 
-| Model Name | Complexity (GFLOPs) | Size (Mp) | Top-1 on ImageNet (%) | Links |
-| ---        | ---                 | ---       | ---                   | ---                   |
-| MobilenetV2 1.0 | 0.66           | 3.50      | 73.0                  | [model template](./mobilenet_v2_w1/template.yaml)
+| Model Name | Complexity (GFLOPs) | Size (Mp) | AVG mAP (%) | AVG Top-1 (%) | AVG Top-5 (%) | Links |
+| ---        | ---        | ---        | ---        | ---        | ---        | ---        |
+| EfficientNet b0 | 0.76 | 4.14 | 92.267 | 90.156 | 98.283 | [model template](./efficientnet_b0/template.yaml) |
+| MobilenetV3 large x1.0 | 0.44 | 4.33 | 92.04	| 89.019 | 97.807 | [model template](./mobilenet_v3_large_1/template.yaml) |
+| MobilenetV3 large x0.75 | 0.308 | 2.84 | 87.59 |	87.59	| 97.17 | [model template](./mobilenet_v3_large_075/template.yaml) |
+| MobilenetV3 small x1.0 | 0.112 | 1.56 | 82.84 | 84.47 | 95.88 | [model template](./mobilenet_v3_small/template.yaml) |
 
-## Datasets
+All of the above metrics were obtained on eleven different datasets, on which an extensive number of researches has been made. To provide a generalized performance metric, we averaged the metrics across all datasets. For additional information about performance on each dataset in comparison with baseline, you can refer to this [spreadsheet](https://docs.google.com/spreadsheets/d/1CV3be-VydEHvWS6GMPduBQBjl46uLq80_GtkeUhsuVg/edit#gid=0).
 
-The model was initially trained on [ImageNet](http://image-net.org/challenges/LSVRC/2012/), but can be trained from scratch or fine tuned to classify arbitrary images.
+The following datasets were used in experiments:
+* [Describable Textures (DTD)](https://www.robots.ox.ac.uk/~vgg/data/dtd/)<sup>1</sup>
+* [Caltech 101](http://www.vision.caltech.edu/Image_Datasets/Caltech101/)<sup>1</sup>
+* [Oxford 102 Flowers](http://www.robots.ox.ac.uk/~vgg/data/flowers/102/)
+* [Oxford-IIIT Pets](https://www.robots.ox.ac.uk/~vgg/data/pets/)
+* [CIFAR100](https://www.cs.toronto.edu/~kriz/cifar.html)
+* [SVHN (w/o additional data)](http://ufldl.stanford.edu/housenumbers/)
+* [Fashion MNIST](https://github.com/zalandoresearch/fashion-mnist)
+* [FOOD101](https://www.kaggle.com/dansbecker/food-101)<sup>1</sup>
+* [SUN397](https://vision.princeton.edu/projects/2010/SUN/)<sup>1</sup>
+* [Birdsnap](http://thomasberg.org/)<sup>1</sup>
+* [Cars Dataset](https://ai.stanford.edu/~jkrause/cars/car_dataset.html)
 
+<sup>1</sup> these datasets have custom splits (random stratified split: 80% - train, 20% - val) and cannot be compared straightforwardly with other research results
+
+Training recipes:
+
+We pretrained all models with imagenet weights and fine-tuned on concrete tasks without freezing any layers.
```

Comment:
specific is better
https://ell.stackexchange.com/questions/13046/difference-between-concrete-and-specific#:~:text=Specific%20is%20approximately%20synonymous%20with,its%20antonym%20is%20usually%20general.&text=Concrete%20is%20approximately%20synonymous%20with,its%20antonym%20is%20usually%20abstract.